docs/_integration-manager/camel-routes/camel-route-aws.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: default
+title: Camel Route Configuration for AWS S3
+nav_order: 2
+parent: File Folder Listener
+---
+# Camel Route Configuration for AWS S3
+
+## Overview
+
+The File Folder Listner can be configured with Camel route and its typically consists of a series of processing steps that are connected in a linear sequence.
+
+## Route Configuration
+
+Note that the File Folder Listener Service must be restarted for any configuration changes to take effect. Make sure you have already completed: File Folder Listener Authorization
+
+**routes-prototype.xml**
+(ProgramDataDirectory)/Actian/FileFolderListener/conf/routes-prototype.xml
+
+NOTE: Multiple routes (i.e. for AWS,Azure, GCP and Local) can be added into a single routes-prototype.xml
+
+Example:
+```
+<?xml version="1.0" encoding="UTF-8"?>
+<routes xmlns="http://camel.apache.org/schema/spring">    
+    <route id="run-jobconfig-with-S3-file" autoStartup="false">
+        <setProperty name="jobConfigId">
+            <constant>102712</constant>
+        </setProperty>
+        <setProperty name="filenameOverride">
```

Comment:
Not sure, whether we need to specify these individual properties we have used in this configurations. 
Example - "jobConfigId" - What is this value? or anything ? Any suggestions?

Reply:
Maybe put a README in this folder that van have as much explanation as you need and then we also need to put in a doc bug to get it further fleshed out in there.

Reply:
Thanks Garrett, Yeah we can add but not sure what to add into the README file. Or is there any other group we need to share the PR for review ?@jsnwgnr  - Could you please help with review once & suggest ?

Reply:
not sure why we would add a README.md to documentation, but there should be a table with names and descriptions of what the individual properties mean and/or do.

Reply:
Done. Please have a look and suggest if any correction needed.